From 69b082936e8d80c7ebf85634e1002280b3c6c062 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 21:02:53 -0500
Subject: [PATCH] Don't treat warnings as errors when building Assimp

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 13854dd353c3..8e2eeadb2dc9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,6 +39,7 @@ ament_vendor(assimp_vendor
     -DASSIMP_BUILD_ASSIMP_TOOLS:BOOL=OFF
     -DASSIMP_BUILD_TESTS:BOOL=OFF
     -DASSIMP_BUILD_SAMPLES:BOOL=OFF
+    -DASSIMP_WARNINGS_AS_ERRORS:BOOL=OFF
     "${ASSIMP_CMAKE_FLAGS}"
     "-DCMAKE_CXX_FLAGS=${ASSIMP_CXX_FLAGS}"
   PATCHES patches
-- 
2.44.0

